API/hermes/hermes.h

```diff
@@ -179,6 +179,9 @@ class HERMES_EXPORT IHermes : public jsi::ICast {
   /// be used by third parties.
   virtual sampling_profiler::Profile dumpSampledTraceToProfile() = 0;
 
+  /// Reset timezone cache
```

Comment:
Can you please add a much more detailed comment explaining what purpose this call serves, uses cases when it should be invoked, etc.